client/components/domains/register-domain-step/index.jsx

```diff
@@ -363,6 +377,25 @@ class RegisterDomainStep extends React.Component {
 		);
 	}
 
+	renderTldButtons() {
+		const { availableTlds, lastFilters: { tlds: selectedTlds } } = this.state;
+		return (
+			<CompactCard className="register-domain-step__tld-buttons">
+				{ availableTlds.slice( 0, 8 ).map( tld => (
```

Comment:
Because `availableTlds` is an object (only when there are TLDs in maintenance, apparently), this line causes an error:
```
Uncaught TypeError: availableTlds.slice is not a function
    at RegisterDomainStep.renderTldButtons (index.jsx:788)
```